src/app/app.module.ts

```diff
@@ -82,7 +82,7 @@ import { DeleteRubricModalComponent } from './components/delete-rubric-modal/del
     imports: [
         appRoutes,
         BrowserModule,
-        NgbModule.forRoot(),
+        NgbModule,
```

Comment:
Was the `forRoot` bit not required? If not, I don't know why I initially added it, other than I must have read you need it somewhere or seen it in an example.